### PR TITLE
change order of items in dependencies part, add link to R packages chapter

### LINKED ISSUE #710: start dependencies section with dependencies advice not technicalities
and link to dependencies chapter of R packages book

### PR DESCRIPTION
**Pull request by maelle**
Fix #710

### CODE REVIEW

File: pkg_building.es.Rmd [hunk 1]
```diff
@@ -446,6 +446,38 @@ Para más detalles sobre las licencias, consulta el libro [R packages](https://r
...
+    Por otro lado, las funciones complejas con muchos casos especiales (por ejemplo, los analizadores sintácticos) requieren considerable testeo y revisión.
+    
+    - Un ejemplo común de esto es devolver "tibbles" usadas por el tidyverse en las funciones del paquete que proporcionan datos.
+      Se puede evitar el uso del paquete **tibble** devolviendo un tibble creado modificando un data.frame de la siguiente manera
+      
+      ```
+      class(df) <- c("tbl_df", "tbl", "data.frame") 
+      ```
+      
+      (Ten en cuenta que este enfoque [no está universalmente aceptado](https://twitter.com/krlmlr/status/1067856118385381377).)
```
Comment (yabellini) [inline pkg_building.es.Rmd:467]: We link to something in twitter, we don't have any example in another place?

File: pkg_building.es.Rmd [hunk 2]
```diff
@@ -446,6 +446,38 @@ Para más detalles sobre las licencias, consulta el libro [R packages](https://r
...
+    La función `%>%` debe importarse de **magrittr** donde está definida, en lugar de **dplyr** que la reexporta y es mucho más pesado.
+  
+  - Algunas dependencias proporcionan nombres de funciones y sintaxis más fáciles de interpretar que los nombres de las funciones y la sintaxis de R base.
+    Si ésta es la razón principal para usar una función en una dependencia pesada, considera incluir el código de R base en una función interna bien nombrada en tu paquete.
+    Consulta, por ejemplo, el [script de R de rlang que proporciona funciones con una sintaxis similar a las funciones de purrr](https://github.com/r-lib/rlang/blob/9b50b7a86698332820155c268ad15bc1ed71cc03/R/standalone-purrr.R).
+  
+  - Si las dependencias tienen funcionalidades redundantes, considera depender de una sola en lo posible.
+  
+  - Puedes encontrar más consejos sobre la gestión de dependencias en [el capítulo "Dependencies: Mindset and Background" (Dependencias: Mentalidad y Contexto) del libro "R packages"](https://r-pkgs.org/dependencies-mindset-background.html) y [en este artículo de Scott Chamberlain (en Inglés)](https://recology.info/2018/10/limiting-dependencies/).
```
Comment (yabellini) [inline pkg_building.es.Rmd:479]:
```suggestion
  - Puedes encontrar más consejos sobre la gestión de dependencias en [el capítulo _"Dependencies: Mindset and Background"_ (Dependencias: Mentalidad y Contexto) del libro [_"R packages"_](https://r-pkgs.org/dependencies-mindset-background.html) y [en este artículo de Scott Chamberlain (en Inglés)](https://recology.info/2018/10/limiting-dependencies/).
```

**Review by yabellini — APPROVED** [review]
Very minor change.

**Review by mpadge — APPROVED** [review]

File: pkg_building.es.Rmd (hunk 1 above)
Comment (maelle) [reply pkg_building.es.Rmd:467]: https://github.com/ropensci/dev_guide/issues/762

**Pull request merged**